macrel/AMP_features.py

```diff
@@ -14,7 +14,14 @@
 
 GROUPS_SA = ['ALFCGIVW', 'RKQEND', 'MSPTHY'] #solventaccess
 GROUPS_HB = ['ILVWAMGT', 'FYSQCN', 'PHKEDR'] # HEIJNE&BLOMBERG1979
-
+colist = ["tinyAA", "smallAA", "aliphaticAA",
```

Comment:
I'm not sure I like this, tbh. It feels more error prone to move the definition further away from the use.

Reply:
already took care of this, and put the list right above the loop using it.